.github/workflows/ci.yml

```diff
@@ -185,3 +185,22 @@ jobs:
           Sincerely,
           The Miri Cronjobs Bot' \
             --user $ZULIP_BOT_EMAIL --api-key $ZULIP_API_TOKEN --site https://rust-lang.zulipchat.com
+      - name: install josh-proxy
+        run: cargo +stable install josh-proxy --git https://github.com/josh-project/josh --tag r22.12.06
+      - name: start josh-proxy
+        run: josh-proxy --local=$HOME/.cache/josh --remote=https://github.com --no-background &
+      - name: setup bot git name and email
+        run: |
+          git config --global user.name 'The Miri Conjob Bot'
+          git config --global user.email 'miri@cron.bot'
+      - name: get changes from rustc
+        run: ./miri rustc-pull
```

Comment:
Potential improvement (but we can leave that for later):
```
./miri toolchain
./miri fmt
git commit -am "rustfmt" || true # allowed to fail since there might be nothing to commit
```

Reply:
Yea, tho I was thinking we could also try to make the regular CI job do that automatically if the fmt job fails. Like for every PR, not just on the sync PR.

Reply:
That sounds fancy... not sure how I feel about auto-pushing to other people's branches, but it might be worth a shot.